### DISCUSSION TITLE
How is terminal packaged in Windows 11?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#16797] [Q&A] [answered]
I have not used Windows 11 so I don't know how terminal is packaged?  I'm trying to help another user with color schemes and I have no idea what he can/can't do.  Terminal is his default for CUI apps.  Is it all (profiles, settings UI, JSON file, color schemes, et c.) pretty much the same as I'm used to with an unzipped-only install on Windows 10?  If not, please give the basics or point me in the direction of a document.  Thanks!

**Comment by DHowett** [comment, marked as answer]
It's almost exactly the same, except that it is registered with the OS in a central location.

We have a [document explaining our "distributions"](https://learn.microsoft.com/en-us/windows/terminal/distributions#distribution-feature-comparison), which breaks down how the features differ between the different kinds of Terminal install. Anything that is *not* on this list can be assumed to be the same between them!

That user you're talking about will fall into the "Packaged" column. You're in the "Unpackaged" column!

**Comment by vefatica** [comment]
Thanks, @DHowett.

**Reply by DHowett** [reply]
Sure thing!
